NewDawn/NewDawn/ProfileViewController/ProfileGNB.storyboard

```diff
@@ -470,11 +470,90 @@
                                 </state>
                                 <connections>
                                     <action selector="nextButtonTapped:" destination="IvN-LX-e3l" eventType="touchUpInside" id="lHm-8G-dFq"/>
+                                    <action selector="nextButtonTapped:" destination="rCc-Xe-gYN" eventType="touchUpInside" id="mBK-ph-46y"/>
```

Comment:
Could you double check this? Why there’s two events of the same reference here? Might be right click on the button and see if you need to remove the unused one?

Reply:
You are right. Because I copied continue button from existing pages. So the old reference is still there. Will remove it.